src/libs/functions/openApp.js

```diff
@@ -113,11 +113,11 @@ const openConnectorInHome = (navigation, connector) => {
  * @returns {Promise}
  */
 export const openApp = (client, navigation, href, app, iconParams) => {
-  const subdomainType = client.capabilities?.flat_subdomains ? 'flat' : 'nested'
+  const subDomainType = client.capabilities?.flat_subdomains ? 'flat' : 'nested'
```

Comment:
nit: `subdomain` is more correct than `subDomain` as it is a single world in english

Reply:
yeah but it's the name in the cozy client function :/